templates/default/sysvinit.service.erb

```diff
@@ -106,6 +109,25 @@ _status_q() {
     _status >/dev/null 2>&1
 }
 
+_wait_for_listening() {
+  echo -n "Waiting for consul daemon to be listening..."
+  for i in `seq 1 30`; do
+    # if ! start-stop-daemon --quiet --stop --test --pidfile $pidfile --user $user; then
+    if ! _status_q; then
```

Comment:
It appeared that both of these checks (`_status_q ` and the previous long version) fails if they are running right after the daemon start. I've reproduced that in Debian 7 via Test Kitchen:
```console
# service consul restart
Starting consul
Waiting for consul daemon to be listening... FAIL: consul process died
```

So, it might be reasonable to add a sleep before the first check.
For example, it could be done by moving `sleep 1` from the end to the beginning of the "for" loop:
```bash
_wait_for_listening() {
    echo -n "Waiting for consul daemon to be listening..."
    for i in `seq 1 30`; do
        sleep 1
        if ! _status_q; then
            echo " FAIL: consul process died"
            return 2
        fi
        if "$exec" info >/dev/null; then
            echo " OK"
            return 0
        fi
        echo -n .
    done
    echo " FAIL: consul process is alive, but is not listening."
    return 2
}
```

P.s. Also, the indentation size is 4 spaces in this file 😉